runtime/criusupport/criusupport.cpp

```diff
@@ -748,7 +783,7 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 
 		vm->extendedRuntimeFlags2 &= ~J9_EXTENDED_RUNTIME2_CRIU_SINGLE_THREAD_MODE;
 
-		toggleSuspendOnJavaThreads(currentThread, FALSE);
+		toggleSuspendOnJavaThreads(currentThread, FALSE, isJdwpEnabled);
```

Comment:
Does this clear the halt flag of non-JDWP threads when JDWP is enabled?

Reply:
Right, the third parameter should always be `FALSE` for this line since it should take the default path of clearing every thread.